repos/cli-utils/src/network/__tests__/getAddresses.js

```diff
@@ -1,5 +1,6 @@
-
+const constants = require('../../constants/constants')
```

Comment:
if you could just add a short comment saying "constants needs unmocked os module" or something, that would be great

Reply:
Will do, but I'm adding it to this PR => https://github.com/simpleviewinc/keg-cli/pull/81
It's a branch off this one, so it will be easier to merge down later